examples/megacli.cpp

```diff
@@ -7180,7 +7117,7 @@ void DemoApp::whyamiblocked_result(int code)
         if (code != 500 && code != 700)
         {
             cout << "Logging out..." << endl;
-            client->locallogout(true);
+            client->locallogout(true, false);
```

Comment:
The SDK behaves differently in this case, keeping sync configs by default --> megacli should behave the same.

Reply:
Fixed in bca6388